File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Facade.Common/Interfaces/ProtocolTemplate.cs
using VecompSoftware.DocSuiteWeb.Data;

namespace VecompSoftware.DocSuiteWeb.Facade.Common.Interfaces
{
    public struct ProtocolTemplate
    {
        public ProtocolStatus Status;
        public ProtocolType Type;
        public DocumentType DocumentType;
        public Location Location;
        public Location AttachLocation;
        public Container Container;
        public Category Category;
        public string Note;
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/Code/ProtocolTemplate.vb
Imports VecompSoftware.DocSuiteWeb.Data

Public Structure ProtocolTemplate
    Public Status As ProtocolStatus
    Public Type As ProtocolType
    Public DocumentType As DocumentType
    Public Location As Location
    Public AttachLocation As Location
    Public Container As Container
    Public Category As Category
    Public Note As String
End Structure